modules/ducktests/tests/ignitetest/utils/ignite_test.py

```diff
@@ -40,3 +44,49 @@ def monotonic():
             so that only the difference between the results of consecutive calls is valid.
         """
         return monotonic()
+
+    def copy_ignite_workdir(self):
+        """
+        Copy work dir from service nodes to the results directory.
+
+        If the test passed, only the default set will be collected. If the the test failed, all logs will be collected.
+        """
+        self.logger.info("copy_ignite_workdir")
+        for service in self.test_context.services:
```

Comment:
you don't need to use `self.test_context.logger`, `Test` has property  `logger` that do the job.

Reply:
fixed